### ISSUE TITLE
Improve disabled styles in File Upload component

### ISSUE THREAD

**Issue by mturley** [patternfly/patternfly#3048]
Discussing with @christiemolloy and @mcarrano, we agreed that the File Upload component looks a little weird when it is disabled:

<img width="558" alt="Screenshot 2020-04-29 16 31 56" src="https://user-images.githubusercontent.com/811963/80645656-6cb82d80-8a39-11ea-9e95-fbc8b4bd8050.png">

Since there is no border on the bottom of the disabled filename input field, it blends with the textarea below into one big box, making the buttons look out of place. @christiemolloy proposed altering the `:disabled` style for this specific class of input fields (only within a file upload), so that it has the bottom border that it currently does when it is `read-only`.

**Comment by mcoker** [comment]
I made a similar note in https://github.com/patternfly/patternfly/pull/3049#issuecomment-621469098.

**Comment by mceledonia** [comment]
Something like this might be more ideal but I can't think of a way to do it that fits well within our initial approach. Is this something that would be a good use case for an override/exception? i.e. if disabled and if an adjacent element is disabled, use this styling?

![image](https://user-images.githubusercontent.com/30935938/81000959-f8a0cf80-8e14-11ea-9a86-8ae40ee21973.png)

**Comment by mcoker** [comment]
After meeting with @mceledonia and @mcarrano, we decided to go forward manually defining a bottom border on disabled elements in the file upload input group.

**Comment by mcarrano** [comment]
@mcoker is this still an issue?

**Comment by mcoker** [comment]
I don't believe so. This is what it looks like when the component is disabled. I want to say this was resolved when we removed the bottom border from disabled control variation buttons. Unless we still want a border between the input group and textarea?
 
<img width="804" alt="Screen Shot 2020-08-06 at 5 19 20 PM" src="https://user-images.githubusercontent.com/35148959/89588520-6cce8300-d809-11ea-995d-079a0871d2a9.png">
